controller-server/src/main/java/com/yahoo/vespa/hosted/controller/deployment/DeploymentStatus.java

```diff
@@ -194,8 +194,14 @@ public boolean hasFailures() {
         return instances.build();
     }
 
-    /** The step status for all steps in the deployment spec of this, in the same order as in the deployment spec. */
-    public List<StepStatus> allSteps() { return allSteps; }
+    /** The step status for all relevant steps in the deployment spec of this, in the same order as in the deployment spec. */
+    public List<StepStatus> allSteps() {
+        List<JobId> firstTestJobs = List.of(firstDeclaredOrElseImplicitTest(systemTest),
```

Comment:
Only these test jobs will be triggered automatically. Undeclared jobs may still be triggered by users, though, but that likely won't happen if they aren't shown here :)

Reply:
When will an implicit job be present here?

Reply:
When there aren't any explicit ones.